client/src/components/categorical/categorical.js

```diff
@@ -42,7 +47,13 @@ class Categories extends React.Component {
         <p> Categorical Metadata </p>
         {_.map(ranges, (value, key) => {
           const isColorField = key.includes("color") || key.includes("Color");
-          if (value.options && !isColorField && key !== "name") {
+          const isSelectableCategory =
+            value.options &&
+            !isColorField &&
+            key !== "name" &&
+            value.numOptions < categorySelectionLimit;
```

Comment:
`value.numOptions < categoryDisplayLimit`

Reply:
I strongly prefer a model where we use the same name for the same concept.   Re-aliasing the command line param seems destined to confuse future readers of the code.